ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -44,42 +41,6 @@ chainSyncBlockServerFollower
     -> m (Follower m blk (WithPoint blk (Serialised blk)))
 chainSyncBlockServerFollower chainDB registry = ChainDB.newFollower chainDB registry getSerialisedBlockWithPoint
 
--- | Chain Sync Server for block headers for a given a 'ChainDB'.
---
--- The node-to-node protocol uses the chain sync mini-protocol with chain
```

Comment:
Would you move this comment sentence to `chainSyncServerForFollower`? It'd be good to put the two driving examples there motivating why `b` does not equal `blk`.